src/main/java/io/securecodebox/persistence/defectdojo/model/BaseModel.java

```diff
@@ -4,11 +4,9 @@
 
 package io.securecodebox.persistence.defectdojo.model;
 
-import java.util.Map;
 import lombok.EqualsAndHashCode;
 
 @EqualsAndHashCode // FIXME: Implement hashCode/equals in inheritance is problematic https://www.artima.com/articles/how-to-write-an-equality-method-in-java (see https://github.com/secureCodeBox/defectdojo-client-java/issues/23)
-// FIXME: Class should be package private because implementation detail
-abstract public class BaseModel {
-  public abstract boolean equalsQueryString(Map<String, Object> queryParams);
+abstract class BaseModel implements Model {
+  // Class can be removed we remove @EqualsAndHashCode.
```

Comment:
if this class can be removed why do you still have it here?

Reply:
Because of the @EqualsAndHashCode. When I remove this, the behavioiur of equals/hashCode of all subclasses may change. Due to missing tests thisis a change which shouldbedone carefully. Thus I will do this in a separate step. Also it is a different [ticket](https://github.com/secureCodeBox/defectdojo-client-java/issues/23).

Reply:
ah ok, the comment was missing the `after`.
I though you meant here that you had already removed it(as you did, just only in this class)